aws/resource_aws_s3_bucket_test.go

```diff
@@ -109,6 +109,11 @@ func TestAccAWSS3Bucket_generatedName(t *testing.T) {
 
 func TestAccAWSS3Bucket_region(t *testing.T) {
 	rInt := acctest.RandInt()
+	partition := testAccGetPartition()
+
+	if partition == "aws-us-gov" {
+		t.Skip("skipping replication tests; govcloud only includes a single region")
```

Comment:
We should probably implement this as a `PreCheck` function (`testAccMultipleRegionsPreCheck(t)`?) rather than duplicating the logic multiple places. This is especially relevant as it will pop up in other services where we have replication testing as well, like RDS.

Reply:
In fact further down the road, coupled with a new `aws_regions` data source that pulls regions from the `endpoints` SDK package, we could have these automatically start running if/when the `endpoints` SDK package gets updated with additional regions in the GovCloud partition. 😉